client/src/components/DownloadModal.js

```diff
@@ -11,16 +11,17 @@ import { useDownloadModal } from 'hooks/useDownloadModal'
 
 // Button and Modal to show when downloading
 export const DownloadModal = ({
-  icon,
+  label,
   resource: initialResource,
-  publicComputedFile: initialPublicComputedFile
+  publicComputedFile: initialPublicComputedFile,
+  icon = null,
+  color = 'brand',
+  disabled = false
```

Comment:
We need to update how we handle `disabled` so that they are the same no matter if we render the "Anchor" version or the "Button" version.

I think this should be enough, and then just use `isDisabled` for both versions.
```javascript
const isDisabled = disabled || publicComputedFile === undefined
```

Or remove `publicComputedFile === undefined` and pass in the disabled attribute manually.

Reply:
Great! See the commit at ea94de48055d089a6f33d49d5a0174dc18dd89bc.